src/editors/containers/ProblemEditor/components/EditProblemView/index.jsx

```diff
@@ -4,31 +4,22 @@ import AnswerWidget from './AnswerWidget';
 import SettingsWidget from './SettingsWidget';
 import QuestionWidget from './QuestionWidget';
 import { EditorContainer } from '../../../EditorContainer';
+import { selectors } from '../../../../data/redux';
+import { useSelector } from 'react-redux';
+import { Editor } from '@tinymce/tinymce-react';
```

Comment:
I will ask a UX question about the level of text-editor flexibility given here. i.e. should we use base tinymce, or use the text editor we just created.

Reply:
@connorhaugh sure I will wait for it. :)

Reply:
@connorhaugh any updates here?

Reply:
I will have an answer tomorrow afternoon after a pre-grooming meeting.